Python/ExcelOutputClass.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""
+Spyder Editor
+
+This is a temporary script file.
+"""
+
+import os
+import xlsxwriter
+import openpyxl
+
+os.chdir(r'\Users\Ben\Documents\GitHub\Ben-Projects')
+
+def make_excel_file(title):
```

Comment:
I would prefer to see this be a generic function.  Although you know what the header will always be for your case, it might not be the same for the next person.  Consider accepting a data collection (likely and array) and looping through that to create the header columns.  It should be the responsibility to the "Caller" to sort their data before handing it to your function (the "Callee")

Try something like this:
def make_excel_file(header_col_names):
	workbook = xlsxwriter.Workbook(title) #make the workbook
 	worksheet = workbook.add_worksheet() #make the worksheet
 	row, col = 0, 0 #we know its the initial row, and we wanna start at the first column
 	for cval in header_col_names: #go through all the values passed to us
 		worksheet.write(row, col, cval) #insert to the proper (row, col) value
 		col += 1 #move to the next column cell